### PR TITLE
Refactor: Object Short Notation

### PR DESCRIPTION
**Pull request by gkal19**
> The other pull request has conflicts

### CODE REVIEW

File: src/ReactTVEntry.js [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
 
 import ReactTVRenderer from './renderer/ReactTVFiberEntry';
+import {default as Platform} from './modules/Platform';
 
 const ReactTV = {
   render: ReactTVRenderer.render,
   unmountComponentAtNode: ReactTVRenderer.unmountComponentAtNode,
 };
 
-export {default as Platform} from './modules/Platform';
```
Comment (raphamorim) [inline src/ReactTVEntry.js]: You still have to export Platform

Comment (raphamorim) [reply src/ReactTVEntry.js]: However it'll fail, Platform is already defined.

**Comment by raphamorim** [comment]
@gkal19 can you squash your commits?

**Comment by raphamorim** [comment]
@gkal19 can you rebase it?

**Comment by gkal19** [comment]
I did the rebase as you asked me however, when trying to squash the commits, more conflicts appear.
```
<<<<<<< refactor/objectShortNotation
    input,
    plugins,
=======
    input: entryPath,
    plugins: plugins,
>>>>>>> master
```

**Comment by raphamorim** [comment]
I'll rewrite all build script then I'm closing it 
Sorry for it 😢

**Comment by gkal19** [comment]
No problem 😢 
I'll keep this branch open.

**Pull request closed without merging**